functions.sh

```diff
@@ -67,17 +67,14 @@ function unsetTee {
 
 # Creates a unique results dir based on date, then links the common
 # results dir name to it.
-
 function setupResultsDir {
-    mkdir -p ${RESULTS_ARCHIVE_DIR}/${DATE}/${RESULTS_SUB_DIR}
+    mkdir -p ${RESULTS_ARCHIVE_DIR}/${DATE}
     # FIXME: do not assume RESULTS_DIR is currently a symlink, and
     # handle appropriately.if not.
-    rm -rf $RESULTS_DIR/${RESULTS_SUB_DIR}
-    mkdir -p $RESULTS_DIR
-    ln -s ${RESULTS_ARCHIVE_DIR}/${DATE}/${RESULTS_SUB_DIR} $RESULTS_DIR/${RESULTS_SUB_DIR}
-    # find a better solution instead of deleting this directory
-    # FIXME: why is it copying ${RESULTS_ARCHIVE_DIR}/${DATE} in $RESULTS_DIR
-    rm -rf $RESULTS_DIR/${DATE}
+    rm -rf $RESULTS_DIR
+    ln -s ${RESULTS_ARCHIVE_DIR}/${DATE} $RESULTS_DIR
+    mkdir -p $TESTING_RESULTS_DIR
+    mkdir -p $BENCHMARK_RESULTS_DIR
```

Comment:
if `benchmark` is not passed to` cronjob`, the benchmark result dir will still be created but empty. That was my motivation when creating the appropriate dir based on the input to `cronjob`. But if having an empty dir is not a problem, I am fine with that

Reply:
I think the empty dir is okay since it (should be) accurate with what the user requested for that particular run (ie. empty if they didn't run benchmarks).  We could just not create the directory at all if they don't run benchmarks, but I was thinking maybe we just keep the code simple since it shouldn't matter.